recipes/micromagneticdata/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "micromagneticdata" %}
+{% set version = "0.12" %}
+{% set sha256 = "010a38a63ff3ca6db32399d45c94c5b66591afac4c49ff9b3d4fb959e1f30b80" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: pip install --no-deps .
+
+requirements:
+  build:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - discretisedfield
+    - oommfodt
+    - ipywidgets
+    - pytest
```

Comment:
Is pytest needed to run the package, or only to test it?

Reply:
`pytest` is a dependency of the package.

Reply:
I think it's only needed for testing, though; it's not listed as a dependency in the setup.py:

https://github.com/joommf/micromagneticdata/blob/8170f3d91215f1e20f8bd93ef827cf51501e4bb3/setup.py#L17-L19

Reply:
It is use for testing - we use it as `import micromagneticdata; micromagneticdata.test()` and we need it as a dependency. It does not show up in `setup.py` because `pytest` is already a dependency of `discretisedfield`. So, we can remove it from here as well.

Reply:
`pytest` is not in the list anymore.